src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -593,7 +593,7 @@ private function addAnnotationsSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('annotations')
                     ->info('annotation configuration')
-                    ->addDefaultsIfNotSet()
+                    ->canBeDisabled()
```

Comment:
Would be better for consistency to use `canBeEnabled`, but that would be a too big BC break. At least with that default, Symfony is going to send a clear exception message about installing `doctrine/annotations`. If not, people will get an error about an unknown `annotation_reader` service.